src/layouts/page/header.tsx

```diff
@@ -1,13 +1,25 @@
 import { siteURL } from "../../lib/url";
 import styles from "./header.module.scss";
 import React from "react";
+import useSwiftType from "../../components/swiftype/use-swiftype";
 
 export function Header() {
+  useSwiftType();
   return (
     <header className={styles.siteHeader}>
       <a id="site-logo" className={styles.siteLogo} href={siteURL.pathname}>
         <img src={`${siteURL.pathname}img/logos/shipengine-connect-logo.svg`} alt="Shipengine Connect" />
       </a>
+
+      {/* SwifType search, with Google Search fallback  */}
+      <form className={`${styles.siteSearchForm} invisible`} method="get" action="https://www.google.com/search">
+        <label htmlFor="site-search" className="visually-hidden">
+          ShipEngine Connect Docs Search
+        </label>
+        <input id="site-search" className="st-default-search-input"
+          name="q" type="search" placeholder="Search" />
+        <input type="hidden" name="as_sitesearch" value="connect.shipengine.com" />
```

Comment:
It's pretty close to [hard-coded anyway](https://github.com/ShipEngine/connect-website/blob/b318ff5478a56f7fabfd189272e950ac0441d6fa/src/lib/url.ts#L7), but it's probably a good practice to use `siteURL` where we can.

```suggestion
        <input type="hidden" name="as_sitesearch" value="${siteURL.hostname}" />
```

Reply:
I just removed the fallback search. It didn't even work anyway. Google can't see the connect subdomain for some reason.